src/fuse_api.pxi

```diff
@@ -313,6 +315,7 @@ def main(workers=None, handle_signals=True):
             session_loop_single()
         else:
             session_loop_mt(workers)
+    t.join()
```

Comment:
Why wait for the thread? If we shutdown anyway, there's no point in delivering remaining notifications.